docs/docs/en/guides/quick-start/install-tuist.md

```diff
@@ -8,18 +8,23 @@ description: Learn how to install Tuist in your environment.
 
 The Tuist CLI consists of an executable, dynamic frameworks, and a set of resources (for example, templates). Although you could manually build Tuist from [the sources](https://github.com/tuist/tuist), **we recommend using one of the following installation methods to ensure a valid installation.**
 
-### Recommended: <a href="https://github.com/jdx/mise">Mise</a> {#recommended-mise}
+### <a href="https://brew.sh">Homebrew</a> {#alternative-homebrew}
 
-Tuist defaults to [Mise](https://github.com/jdx/mise) as a tool to deterministically manage and activate versions of Tuist.
-If you don't have it installed on your system,
-you can use any of these [installation methods](https://mise.jdx.dev/getting-started.html).
-Remember to add the suggested line to your shell, which will ensure the right version is activated when you choose a Tuist project directory in your terminal session.
+You can install Tuist using [Homebrew](https://brew.sh) and [our formulas](https://github.com/tuist/homebrew-tuist):
+
+```bash
+brew tap tuist/tuist
+brew install --formula tuist
+brew install --formula tuist@x.y.z
+```
```

Comment:
I think mise should be mentioned first.
I agree it's important to not make look brew as non-official, but the way I would recommend is still mise, and so I would mention it first as you usually follow this kind of guides from top to bottom

Reply:
That makes sense. I updated it.